docs/tutorials/using-jsx-in-apostrophe.md

```diff
@@ -173,17 +177,18 @@ import App from './jsx-components/App.jsx';
 export default () => {
   apos.util.widgetPlayers.reactWeather = {
     selector: '[data-react-weather-widget]',
-    player: function(el) {
+    player: function (el) {
```

Comment:
Method syntax is nicer here (an actual class is not required):

```
player(el) { ... }
```

Reply:
Changed.